addons/account/wizard/wizard_tax_adjustments.py

```diff
@@ -20,10 +25,11 @@ def _get_default_journal(self):
                                         domain="[('deprecated', '=', False), ('is_off_balance', '=', False)]")
     amount = fields.Monetary(currency_field='company_currency_id', required=True)
     adjustment_type = fields.Selection([('debit', 'Applied on debit journal item'), ('credit', 'Applied on credit journal item')], string="Adjustment Type", required=True)
-    tax_report_line_id = fields.Many2one(string="Report Line", comodel_name='account.tax.report.line', required=True, help="The report line to make an adjustment for.")
+    tax_report_line_id = fields.Many2one(string="Report Line", comodel_name='account.tax.report.line', required=True, help="The report line to make an adjustment for.",
+                                         domain="[('tag_name', '!=', None), ('report_id.country_id', 'in', fiscal_countries)]")
     company_currency_id = fields.Many2one('res.currency', readonly=True, default=lambda x: x.env.company.currency_id)
     report_id = fields.Many2one(string="Report", related='tax_report_line_id.report_id')
-
+    fiscal_countries = fields.Many2many('res.country', compute=_compute_fiscal_countries, readonly=True)
```

Comment:
Relational fields should end with `_id` (many2one) or `_ids` (x2many)
https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#symbols-and-conventions